test/index.js

```diff
@@ -40,3 +41,14 @@ describe("check if date is invalid", function () {
     DateValidator.isValidDate("AA/BB/2017", "DD/MM/YYYY").should.equal(false);
   });
 });
+
+describe("check if date is in the past or not", function() {
+  it("returns false for past check on a given random date in the future", function () {
+    DateValidator.isDatePast(moment().add(5, 'days'), "YYYY-MM-DD").should.equal(false);
+  });
+
+  it("returns true for past check on given date random date in the past", function () {
+    console.log(Math.floor(Math.random()) * 100);
```

Comment:
Do we need this console log here?
If its not required you can remove it, we would like to keep the code cleaner.

Reply:
Er sorry. I totally forgot that haha